ligi/SurvivalManual
# Update Icon Name and Path from Portrait to Psychology

### Fix

Update the file name and path data of the `ic_portrait` drawable to `ic_psychology` to match the [Material icon for psychology](https://fonts.google.com/icons?selected=Material%20Icons%3Apsychology%3A) and close #212.  See the screenshots below for illustration.

![212_update_icon_psychology](https://user-images.githubusercontent.com/3827611/132738073-97a3f5db-9a96-4b92-83cd-3c10983d6a89.png)

### Test
1. Open navigation drawer.
2. Notice ***Psychology*** icon matches [Material icon](https://fonts.google.com/icons?selected=Material%20Icons%3Apsychology%3A).
